src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -1897,6 +1904,11 @@ int wm_vuldet_linux_oval_vulnerabilities(sqlite3 *db, agent_software *agents_it,
             version = src_version;
         }
 
+        // We discard the package if its vendor is a external vendor.
+        if (agents_it->dist == FEED_REDHAT && vendor && wm_vuldet_compare_vendors(vendor)) {
+            continue;
```

Comment:
Include a debug2 message here to know the package has been discarded, including the vendor